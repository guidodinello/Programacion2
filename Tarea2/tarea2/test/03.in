# Implmentar insertarAntes
insertarAlFinal (5,2.8)
inicioCadena
insertarAntes (3,6.7)
infoCadena
inicioCadena
infoCadena
siguiente
infoCadena
insertarAntes (4,3.1)
infoCadena
inicioCadena
infoCadena
siguiente
infoCadena
insertarAntes (8,2.9)
infoCadena
inicioCadena
infoCadena
insertarAntes (1,7.2)
infoCadena
inicioCadena
infoCadena
siguiente
infoCadena
siguiente
infoCadena
siguiente
infoCadena
siguiente
infoCadena
Fin