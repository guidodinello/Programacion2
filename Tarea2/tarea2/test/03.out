1>#  Implmentar insertarAntes.
2>Insertado al final.
3>loc al inicio.
4>Insertado antes de loc.
5>(5,2.80)
6>loc al inicio.
7>(3,6.70)
8>loc al siguiente.
9>(5,2.80)
10>Insertado antes de loc.
11>(5,2.80)
12>loc al inicio.
13>(3,6.70)
14>loc al siguiente.
15>(4,3.10)
16>Insertado antes de loc.
17>(4,3.10)
18>loc al inicio.
19>(3,6.70)
20>Insertado antes de loc.
21>(3,6.70)
22>loc al inicio.
23>(1,7.20)
24>loc al siguiente.
25>(3,6.70)
26>loc al siguiente.
27>(8,2.90)
28>loc al siguiente.
29>(4,3.10)
30>loc al siguiente.
31>(5,2.80)
32>Fin.
